pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Problem
+- *Description of why you made this PR*
+
+## Solution
+- *And how you fix that*
+
+## PR Status
+- [ ] Work finished.
```

Comment:
A very minor wording suggestion: I would interpret "work" as "every checkbox", and would suggest to use "coding" here. What do you think?

Reply:
Honestly I think that we can't expect that all the check boxes are going to be validated for each PR.
Myself, I don't test an upgrade each time, now I trust the CI on this point.
And for an minor upgrade without consequences, I'll probably not even try the fix, and again trust the CI.

So, for me, "Work finished" means that the PR is complete. That's, for example, not true for #33.
And, again for #33, that doesn't mean code, but finding the real cause of the issue.

The idea by adding those check boxes is really to know what has been done by the dev who add the PR. And what hasn't.

Reply:
OK, then what difference between "work finished" and "can be reviewed and tested"?

Reply:
Hum, yes I'm not sure that I can say what would be the difference...
Which it's obviously a problem...

But, there's also the case of our testing release, which are finished, but not obviously opened to review yet.
It can be that. But, obviously, if it's not clear, it's useless !